climada/entity/measures/base.py

```diff
@@ -81,33 +83,86 @@ class Measure():
         cost of risk transfer
     """
 
-    def __init__(self):
-        """Empty initialization."""
-        self.name = ''
-        self.haz_type = ''
-        self.color_rgb = np.array([0, 0, 0])
-        self.cost = 0
+    def __init__(
+        self,
+        name: str = "",
+        haz_type: str = "",
+        color_rgb: Optional[np.ndarray] = None,
+        cost: float = 0,
+        hazard_set: str = NULL_STR,
+        hazard_freq_cutoff: float = 0,
+        exposures_set: str = NULL_STR,
+        imp_fun_map: str = NULL_STR,
+        hazard_inten_imp: Tuple[float, float] = (1, 0),
+        mdd_impact: Tuple[float, float] = (1, 0),
+        paa_impact: Tuple[float, float] = (1, 0),
+        exp_region_id: Optional[list] = None,
+        risk_transf_attach: float = 0,
+        risk_transf_cover: float = 0,
+        risk_transf_cost_factor: float = 1,
```

Comment:
The color is not very relevant unless for plotting, so maybe we should move to the last place. But really not important.

```suggestion
        self,
        name: str = "",
        haz_type: str = "",
        cost: float = 0,
        hazard_set: str = NULL_STR,
        hazard_freq_cutoff: float = 0,
        exposures_set: str = NULL_STR,
        imp_fun_map: str = NULL_STR,
        hazard_inten_imp: Tuple[float, float] = (1, 0),
        mdd_impact: Tuple[float, float] = (1, 0),
        paa_impact: Tuple[float, float] = (1, 0),
        exp_region_id: Optional[list] = None,
        risk_transf_attach: float = 0,
        risk_transf_cover: float = 0,
        risk_transf_cost_factor: float = 1,
        color_rgb: Optional[np.ndarray] = None
```